Assets/Scripts/ObjectPool/Cube.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+
+namespace ObjectPool
+{
+    public class Cube : MonoBehaviour, IObjectPoolable
```

Comment:
create a folder Sample in ObjectPool and move Cube (rename to CubeExample) there